### ISSUE TITLE
Can't get the patch working on the GOG version of Ultima Underworld 1

### ISSUE THREAD

**Issue by TheWeekendSlice** [JohnGlassmyer/UltimaHacks#10]
Hi everyone,

I can't seem to get the Ultima Hacks patch to work with the GOG version of Ultima Underworld I.

I have patched the game using the online patcher, I then replaced the original executable with the patched one. Just as mentioned in the instructions,.

However I can't seem to recreate a game.gog image (tried with 7zip or an iso maker) that works with the GOG provided dosbox. Dosbox starts up but when I try to launch the game it just closes back down.

I also don't have any idea how to get the GOG provided dosbox to point at the folder in which I unpacked the original game.gog image (and in this folder is where the patched executable exists).

Please help.

Thank you!

**Comment by JohnGlassmyer** [comment]
If I remember correctly, one of the dosbox conf files that GOG installs with UW1 (or with UW2) contains an IMGMOUNT command that mounts the game.gog file as a virtual drive within the DOSBox environment. You can probably comment out that IMGMOUNT command and replace it with a regular MOUNT command that instead mounts a directory containing the extracted contents of game.gog as that virtual drive.

Does that help?

**Comment by TheWeekendSlice** [comment]
I have tried to get the GOG dosbox to point at the folder where I unpacked the original game.gog, but it doesn't want to load the game. I edited the config files of the GOG dosbox and no matter what I try, it refuses to load the game.

The GOG dosbox initially activates a launcher in dosbox, this is a menu from which you can launch the game, go to the game's dos settings or exit dosbox back to windows. The option to launch the game is set to point to a file named uw.bat. I tried getting it to point to the hacked uw.exe and even to the non-hacked uw.exe and it simply doesn't load the game.

I installed my own copy of Dosbox (from their website) and I can get the hacked game to load in that, but then it doesn't have any sound for some reason.

Oh, and let me just say, that the hack does work, mouselook works and is great, I am just having a hard time getting the hack to work with the GOG version of game. When using the GOG provided dosbox, it just doesn't want to load the hacked exectuable. When using a separate, fresh install of Dosbox (from their website) the game loads, the hack works, but I get no sound.

**Comment by JohnGlassmyer** [comment]
Were you able to find the file `dosboxULTIMA1_single.conf` in the GOG UW installation, and the imgmount command within that file? It should be straightforward to apply what I wrote in my previous comment once you have located that. I think you should not need to make other edits to the config.

**Comment by TheWeekendSlice** [comment]
YES! Success, I got it to work by trying a new ISO making program: http://www.winiso.com/products/winiso-free.html
1) I used 7zip to extract the files from the original game.gog
2) I used the online hack patcher to patch uw.exe
3) I replaced the original uw.exe with the hacked one (I had to rename the hacked executable to uw.exe)
4) I built an iso in the program I linked to above. The iso was saved as game.iso but I renamed it to game.gog
5) I replaced the original game.gog with my version of game.gog that contains the hacked uw.exe

Game now works, mouselook works, sound also works.
